plugins/modules/ome_application_certificate.py

```diff
@@ -209,45 +243,61 @@ def format_csr_string(csr_string):
     return formatted_csr
 
 
+def get_ome_version(rest_obj):
```

Comment:
Check if there a util for this already, if not move this to utils.

Reply:
Done